package.json

```diff
@@ -25,39 +25,41 @@
   "license": "Apache-2.0",
```

Comment:
I may have missed how this works, but where/how would I run the typescipt compiler to generate JS from any TS?